<launch>
    <node name='adroc_diver' pkg='adroc_diver' type='autonomous_operator_configuration_deploy_planA.py' output='screen'> 
        <param name='x_error_tolerance' value='0.15'/>
        <param name='y_error_tolerance' value='0.15'/>
        <param name='pd_error_tolerance' value='0.1'/>
        <param name='drp_active_timeout' value='1'/>
    </node>
</launch>
